reacthookspring/dockerfile

```
@@ -1,7 +1,7 @@
 FROM openjdk:11-jre-slim-buster
 # copy application JAR (with libraries inside)
 
-ADD application.jar /opt/application.jar
+ADD application/target/application-0.0.1-SNAPSHOT.jar /opt/application.jar
```

This could lead to an error when changing the version. Maybe something like this?
```suggestion
ADD application/target/application-*.jar /opt/application.jar
```